atlas/getting-started/deploy-from-existing-repo.mdx

```diff
@@ -0,0 +1,112 @@
+---
+slug: /atlas/getting-started/deploy-from-existing-repo
+title: Deploy From An Existing Repository
+description: Deploy an Atlas app from an existing repository.
+---
+
+Now that you have [chosen to deploy from an existing repository](./blueprint-repository), you can select the repository and deploy it to Atlas. Pushing to your main branch in GitHub will then deploy updates to Atlas automatically.
+
+Atlas supports Node.js applications using popular frameworks like [Faust.js](https://faustjs.org/), [Next.js](https://nextjs.org/), [Gatsby](https://www.gatsbyjs.com/), [Nuxt](https://nuxtjs.org/), [SvelteKit](https://kit.svelte.dev/), etc.
+
+## Create Atlas App
+
+### 1. Prepare your app for deployment
```

Comment:
This part of the flow feels much improved. Good choice for a heading, as this orients the user around pre-deployment work they should do _before_ making the GitHub connection, instead of having the steps lumped together as in the previous draft.